src/controller/cve.controller/index.js

```diff
@@ -346,7 +322,7 @@ router.post('/cve/:id/cna',
     }
   }
   #swagger.responses[200] = {
-    description: 'The requested CVE record is returned.',
+    description: 'The requested CVE record is returned',
     content: {
       "application/json": {
         schema: { $ref: '/schemas/cve/create-cve-record-response.json' }
```

Comment:
Response JSON should match 
{
    "message": "CVE-2005-0003 record was successfully created.",
    "created": {
        "dataType": "CVE_RECORD",
        "dataVersion": "5.0",
        "cveMetadata": {
            "cveId": "CVE-2005-0003",
            "assignerOrgId": "b8e70571-7ecc-4e74-b5bb-cd168f4198f7",
            "state": "PUBLISHED",
            "assignerShortName": "mitre",
            "requesterUserId": "00c39ac3-a55c-473b-9b3e-bf40ab7d8d33",
            "dateReserved": "2014-04-27T00:00:00.000Z",
            "datePublished": "2022-05-12T16:58:59.564Z"
        },
        "containers": {
            "cna": {
                "x_a": {
                    "$b": "c"
                },
                "affected": [
                    {
                        "product": "p",
                        "vendor": "v",
                        "versions": [
                            {
                                "version": "1.2",
                                "status": "affected"
                            }
                        ]
                    }
                ],
                "descriptions": [
                    {
                        "lang": "en",
                        "value": "v p before 1.2 is insecure."
                    }
                ],
                "problemTypes": [
                    {
                        "descriptions": [
                            {
                                "description": "insecurity",
                                "lang": "en"
                            }
                        ]
                    }
                ],
                "providerMetadata": {
                    "orgId": "b8e70571-7ecc-4e74-b5bb-cd168f4198f7",
                    "shortName": "mitre"
                },
                "references": [
                    {
                        "url": "https://example.com"
                    }
                ]
            }
        }
    }
}